#Service used to send velocity-radius information to the ArcDriveController

#Tangential Velocity To Maintain
float64 x_velocity

#Radius of arc to create
float64 y_velocity
